### PR TITLE
Support AWS EKS

### PR DESCRIPTION
**Pull request by Tang8330**
As per title.

Need to pass in:
1. source.aws_cluster_auth (bool)
2. source.aws_access_key_id
3. source.aws_secret_access_key
4. source.aws_region
5. source.aws_cluster_name

Also adding some other fixes like:
* Typos
* Correcting Docker Labels
* Fixing Make to read VERSION

### CODE REVIEW

**Comment by Typositoire** [comment]
Concourse: Version bump did not happen. v1.29.1 is the same as v1.29.1

**Comment by Typositoire** [comment]
Concourse: Version bump did not happen. v1.29.1 is the same as v1.29.1

**Comment by Typositoire** [comment]
Concourse: Version bump found.

**Comment by Typositoire** [comment]
Concourse: Version bump found.

**Comment by Typositoire** [comment]
Concourse: Version bump found.

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,12 @@
-FROM alpine/helm:3.10.2
-LABEL maintainer "Yann David (@Typositoire) <davidyann88@gmail>"
+FROM --platform=linux/amd64 alpine/helm:3.10.2
```
Comment (Tang8330) [inline Dockerfile:1]: This fix is for Macbook M1/M2s

**Comment by Tang8330** [comment]
@Typositoire Bump, can we get this reviewed please 🙏 . I also saw https://github.com/Typositoire/concourse-helm3-resource/issues/102, and I'm happy to help maintain this repo. Would be great to get a couple more maintainers too!

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.29.1
+1.29.2
```
Comment (Typositoire) [inline VERSION:1]: That would be 1.30.0 since we're adding a feature

**Review by Typositoire — COMMENTED** [review]
@ me when the version is updated! Sorry I'm struggling with notifications ahah

File: VERSION (hunk 2 above)
Comment (Tang8330) [reply VERSION]: I bumped it to 1.31.0 since 1.30 is on master: https://github.com/Typositoire/concourse-helm3-resource/blob/main/VERSION

**Comment by Tang8330** [comment]
@Typositoire Version bumped

**Comment by Typositoire** [comment]
Concourse: Version bump found.

**Comment by Typositoire** [comment]
Concourse: Version bump found.

**Comment by Typositoire** [comment]
Ehhh h won't let me merge on the app, says im not an admin lol. Give me like 10 minutes

**Review by Typositoire — APPROVED** [review]

**Comment by Typositoire** [comment]
@Tang8330 What's the best place to talk about helping to maintain this project ?

**Pull request merged**